using-mobile-extensions/mobile-core/identity/identity-event-reference.md

```diff
@@ -2,68 +2,248 @@
+| `advertisingidentifier` | String         | Yes          | The value of the Advertising Identifier for the mobile device that needs to be set in the SDK. This key will be populated when `setAdvertisingIdentifier` API is called. |
+| `pushidentifier`        | String         | Yes          | The value of the Push Identifier that needs to be set. This key will be populated when `setPushIdentifier` API is called |
 
-### Visitor get IDs request
+#### Event data example
 
-This event is generated by customers that use the `visitorGetIds` API.
+Set Advertising Identifier request
 
-There are no key-value pairs in this event.
+```json
+{
+    "advertisingidentifier": "advertisingIdExample"
+}
+```
+
+#### Event data example
+
+Set Push Identifier request
+
+```json
+{
+    "pushidentifier": "pushIdExample"
+}
+```
 
 ## Events dispatched
 
-The following event is dispatched by the Identity extension:
+### Identity response identity
 
-### Visitor MID response
+This event is a response from the Identity extension to the public API callbacks and it will be generated in response to an Identity Request Identity event when:
 
-This event is generated by the visitor ID extension as a response to the request for `visitorMarketingCloudID` API.
+- Retrieving the Experience Cloud ID (MID) 
+- Retrieving the custom visitor identifiers
+- Synchronizing a visitor identifier or a list of identifiers with the Adobe Experience Cloud ID Service is complete
+- The result for the append to URL request is ready
+
+#### Event details
+
+| **Event Type**               | **Event Source**                       | **Paired** | **Paired Event**                                             |
+| :--------------------------- | :------------------------------------- | :--------- | :----------------------------------------------------------- |
+| com.adobe.eventType.identity | com.adobe.eventSource.responseIdentity | Yes        | [Identity Request Identity Event](#identity-request-identity) |
 
 #### Data payload definition
 
 Here are the key-value pairs in this event:
 
-| **Key or Key Type** | **Value Type** | **Optional** | **Description** |
-| :--- | :--- | :--- | :--- |
-| `mid (String)` | String | false | Auto Generated MID |
-| `advertisingIdentifier (String)` | String | false | Blank if not present |
-| `pushIdentifier (String)` | String | false | Blank if not present |
-| `blob (String)` | String | false | Blank if not present |
-| `locationHint (String)` | String | false | Blank if not present |
-| `visitorIDsList (String)` | List | false | Default: Empty List |
-| `lastSync (String)` | Long | false | default = 0 |
+| **Key**                 | **Value Type**            | **Optional** | **Description**                                              |
+| :---------------------- | :------------------------ | :----------- | :----------------------------------------------------------- |
```

Comment:
looks like the sentences in the "Description" column are missing periods. Please add.